### ISSUE TITLE
Nan llm loss when finetuning EVE-7B-pretrain on llava_v1_5_mix665k

### ISSUE THREAD

**Issue by HYZ17** [baaivision/EVE#11]
I am trying to finetune EVE-7B-pretrain on [llava_v1_5_mix665k](https://huggingface.co/datasets/liuhaotian/LLaVA-Instruct-150K/blob/main/llava_v1_5_mix665k.json) using the script [here](https://github.com/baaivision/EVE/blob/main/scripts/eve/eve7b_finetune.sh) . And the output indicates that some of the `llm_loss` are equal to Nan value, which leads to Nan value of `all_loss`.  In addition, there are many warnings about the "tokenization mismatch". Is this normal? Thank you for your reply. 

<img width="629" alt="image" src="https://github.com/user-attachments/assets/9d10364d-be68-4b93-8d2f-5fa40e49301e">

**Comment by Paranioar** [comment]
It's unusual. Are you trying other base LLM models, using different tokenizers and processors, or perhaps using the wrong model and data sources? We encountered a few "tokenization mismatch" warnings but did not experience llm_loss = NAN.
